templates/image-builder.yml

```diff
@@ -214,7 +214,7 @@ objects:
         initContainers:
         - name: image-builder-migrate
           image: "${IMAGE_NAME}:${IMAGE_TAG}"
-          command: [ "/app/image-builder-migrate-db" ]
+          command: [ "/opt/migrate/tern", "migrate", "-m", "${MIGRATIONS_DIR}" ]
```

Comment:
Didn't you delete `MIGRATIONS_DIR` later in this file?

Reply:
It's still a template variable; but idk if it makes much sense to have it as a variable in the first place tbh, since we always copy the migrations to /app/migrations in the container.

Reply:
I just hardcoded the path